### PR TITLE
Refactor utility methods out of oneoff/training_curve

### PR DESCRIPTION
**Pull request by sethtroisi**
I'm working on several oneoff scripts that will benefit from this code reuse

Additionally I plan to add some dataframe checkpointing code here to speed up training_curve.

### CODE REVIEW

File: oneoffs/training_curve.py [hunk 1]
```diff
@@ -14,31 +14,23 @@
...
 import pandas as pd
 import tensorflow as tf
-import sgf
-import pdb
+import oneoff_utils
 
 from tqdm import tqdm
-from sgf_wrapper import sgf_prop
-from gtp_wrapper import make_gtp_instance, MCTSPlayer
+from gtp_wrapper import MCTSPlayer
```
Comment (amj) [inline oneoffs/training_curve.py]: where is this import used?

Comment (sethtroisi) [reply oneoffs/training_curve.py]: Line 70.

**Comment by sethtroisi** [comment]
Autopep8 run.

**Pull request merged**